### PR TITLE
Avoid translation - Language toggle + Splash page language selection buttons

### PR DESCRIPTION
**Pull request by Ricokola**
- Creation and addition of the documentation and working example for the Language Toggle link.
- Update the language toggle link and the language selection buttons from the splash page with the `translation="no"` attribute.

### CODE REVIEW

File: templates/splash/api-en.html [hunk 1]
```diff
@@ -40,6 +41,8 @@ <h2>Known instances</h2>
 
 <h2>Version history</h2>
 <dl>
+	<dt>Version v2.2 - 2022-10-31</dt>
+	<dd>Addition of the <code>translation</code> attribute set to "no" to each language selection buttons: English and Français</dd>
```
Comment (GormFrank) [inline templates/splash/api-en.html:45]: Name of the attribute is `translate`:
```suggestion
	<dd>Addition of the <code>translate</code> attribute set to "no" to each language selection buttons: English and Français</dd>
```

File: sites/language/includes/languagetoggle.html [hunk 2]
```diff
@@ -4,7 +4,7 @@ <h2 class="wb-inv">{{ i18nText-language }}</h2>
 		<li>
 			<a lang="{{ i18nText-altLang }}" hreflang="{{ i18nText-altLang }}" href="{{ page.altLangPage }}">
 				<span class="hidden-xs">{{ i18nText-altLanguage }}</span>
```
Comment (GormFrank) [inline sites/language/includes/languagetoggle.html:6]: This one should also have the attribute:
```suggestion
				<span class="hidden-xs" translate="no">{{ i18nText-altLanguage }}</span>
```

**Comment by duboisp** [comment]
* [ ] Build a prototype for peers review
* [ ] Code review, build and local testing

**Comment by duboisp** [comment]
Here a prototype with this change: https://universallabs.org/wet/labs/pr2045/index-en.html
base on commit: b75ad8c

**Review by delisma — APPROVED** [review]

**Comment by duboisp** [comment]
Tested and reviewed. All goods.

**Comment by GormFrank** [comment]
Approved internally by PP management :)

**Review by GormFrank — APPROVED** [review]

**Pull request merged**